tendrl/node_agent/node_sync/node_status_update.py

```diff
@@ -0,0 +1,21 @@
+import etcd
+
+
+def update():
+    try:
+        nodes = NS._int.client.read("/nodes")
+    except etcd.EtcdKeyNotFound:
+        return
+
+    for node in nodes.leaves:
+        node_id = node.key.split('/')[-1]
+        try:
+            NS._int.client.read(
+                "/nodes/{0}/NodeContext/status".format(node_id)
+            )
+        except etcd.EtcdKeyNotFound:
```

Comment:
This read and then write is not atomic

use https://github.com/Tendrl/node-agent/blob/d5f851b07da64edbf2676bdc5f40d17342bec29f/tendrl/node_agent/node_sync/sds_detect.py#L61

instead of reading and then writing,  catch etcd.EtcdAlreadyExist

Reply:
ack